src/IconButtonCatalog.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import ComponentCatalogPanel from './ComponentCatalogPanel.js';
+import {MDCIconButtonToggle} from '@material/icon-button';
+import {MDCRipple} from '@material/ripple';
+
+import './styles/IconButtonCatalog.scss';
+
+const IconToggleCatalog = () => {
+  const description = 'Icons are appropriate for buttons that allow a user to take actions or ' +
+                      'make a selection, such as adding or removing a star to an item. ';
+  return (
+    <ComponentCatalogPanel
+      hero={<IconButtonToggle/>}
+      title='Icon Button'
+      description={description}
+      designLink='https://material.io/go/design-buttons'
+      docsLink='https://material.io/components/web/catalog/buttons/icon-button-buttons/'
```

Comment:
Shouldn't this be `.../catalog/buttons/icon-buttons/` to match the path given in the icon-button readme front-matter?

Reply:
https://material.io/develop/web/catalog/buttons/icon-buttons/ doesn't point to anything right now.

Reply:
Neither does the link that's currently in this PR. The one I posted _will_ point to something as soon as the next release happens and material.io is updated.

I don't think there's any perfect order of operations, since one thing has to happen before the other (the material.io update and the catalog update). We should probably aim for material.io to update first since that is probably the lengthier process.